model-archiver/model_archiver/arg_parser.py

```diff
@@ -143,4 +143,13 @@ def export_model_args_parser():
             " packages.",
         )
 
+        parser_export.add_argument(
+            "-c",
+            "--config-file",
+            required=False,
+            type=str,
+            default=None,
+            help="Path to a yaml file containing model configuration eg. batch_size.",
```

Comment:
I'm still a bit confused as to the scope of each config we have, if we're going forward with this to set batch sizes we should also set deprecation warnings if people try to set batch sizes from config.properties